pymongo/daemon.py

```diff
@@ -74,21 +78,28 @@ def _spawn_daemon(args):
     # we spawn the mongocryptd daemon process.
     def _spawn(args):
         """Spawn the process and silence stdout/stderr."""
-        with open(os.devnull, 'r+b') as devnull:
-            return subprocess.Popen(
-                args,
-                close_fds=True,
-                stdin=devnull, stderr=devnull, stdout=devnull)
-
+        try:
+            with open(os.devnull, 'r+b') as devnull:
+                return subprocess.Popen(
+                    args,
+                    close_fds=True,
+                    stdin=devnull, stderr=devnull, stdout=devnull)
+        except FileNotFoundError as exc:
+            raise Exception(
+                f'Failed to start {args[0]}: is it on your PATH: {exc}')
 
     def _spawn_daemon_double_popen(args):
         """Spawn a daemon process using a double subprocess.Popen."""
         spawner_args = [sys.executable, _THIS_FILE]
         spawner_args.extend(args)
-        temp_proc = subprocess.Popen(spawner_args, close_fds=True)
-        # Reap the intermediate child process to avoid creating zombie
-        # processes.
-        _popen_wait(temp_proc, _WAIT_TIMEOUT)
+        try:
+            temp_proc = subprocess.Popen(spawner_args, close_fds=True)
+            # Reap the intermediate child process to avoid creating zombie
+            # processes.
+            _popen_wait(temp_proc, _WAIT_TIMEOUT)
+        except FileNotFoundError as exc:
+            raise Exception(
+                f'Failed to start {args[0]}: is it on your PATH: {exc}')
```

Comment:
We can revert the changes to this method. It's spawning this file itself which should always exist.